src/bodyScrollLock.js

```diff
@@ -77,8 +77,10 @@ const setOverflowHidden = (options?: BodyScrollOptions) => {
       const scrollBarGap = window.innerWidth - document.documentElement.clientWidth;
 
       if (reserveScrollBarGap && scrollBarGap > 0) {
+        const computedBodyPaddingRight = getComputedStyle(document.body).getPropertyValue('padding-right');
```

Comment:
The call to `getPropertyValue` will return a string, so you'll get something like `"0px15px"`
```suggestion
        const computedBodyPaddingRight = parseInt(getComputedStyle(document.body).getPropertyValue('padding-right'), 10);
```